Mathlib/Algebra/DualNumber.lean

```diff
@@ -81,8 +81,17 @@ theorem snd_mul [Semiring R] (x y : R[ε]) : snd (x * y) = fst x * snd y + snd x
 @[simp]
 theorem eps_mul_eps [Semiring R] : (ε * ε : R[ε]) = 0 :=
   inr_mul_inr _ _ _
+
 #align dual_number.eps_mul_eps DualNumber.eps_mul_eps
 
+@[simp]
+theorem inv_eps [DivisionRing R] : (ε : R[ε])⁻¹ = (0 : R[ε]) :=
+  TrivSqZeroExt.inv_inr
+
+@[simp]
+protected theorem inv_one [DivisionRing R] : (inl 1 : R[ε])⁻¹ = (inl 1 : R[ε]) := by
+  rw [TrivSqZeroExt.inv_inl one_ne_zero, inv_one]
```

Comment:
This one should be in the `TrivSqZeroExt` file, stated as `(1 : tsze R M)⁻¹ = 1`; and similarly for `0`.